registry/quilt_server/views.py

```diff
@@ -1645,8 +1645,15 @@ def disable_user():
 
     data = request.get_json()
     username = data.get('username')
+    user_regex = re.compile(r"^[^\d\W]\w*\Z", re.UNICODE)
```

Comment:
post-commit, pls ping @dimaryaz to see how he likes to test things like this, where the regexp could be shared with the core or not:
- if it's shared with the core, then cleaner reuse
- if it's not shared with the core, then we (in theory) also test the regexp

again, IMHO pls commit/merge first then followup about this detail.

Reply:
Ok, got it